tailwind.config.ts

```diff
@@ -71,6 +72,17 @@ export default {
         md: '0px 4px 7px #00000040',
         lg: '0px 4px 10px 0px #00000040',
       },
+      screens: {
```

Comment:
I thought the new way to do this was via css:

```
@import "tailwindcss";
@theme {
  --breakpoint-xs: 30rem;
  --breakpoint-2xl: 100rem;
  --breakpoint-3xl: 120rem;
}
```

https://tailwindcss.com/docs/responsive-design#using-custom-breakpoints

Reply:
me too. it was working in the dev just not prod

Reply:
I'll approve, but let's pair up on it when we get a chance.